feat(docs): add documentation about the new namespaces tab (#1583)
kestra-io/docs: content/docs/04.workflow-components/02.namespace.md

Additionally, you can organize your code on a namespace-level using the embedded Code editor and [Namespace Files](/docs/developer-guide/namespace-files), with the option to [sync those files from Git](/docs/developer-guide/git):

![namespace_mgmt_2](/docs/concepts/namespace_2.png)

## Namespace Tab

> Starting 0.18.0, Kestra has introduced the **Namespaces** tab in Kestra UI for OSS. In this tab, you can see all the namespaces associated with the different flows in Kestra. You can open the details about any namespace by clicking on the details button to the right of that namespace.

Reviewer: ```suggestion
Starting 0.18.0, Kestra has introduced the **Namespaces** tab in the Kestra UI for OSS. In this tab, you can see all the namespaces associated with the different flows in Kestra. You can open the details about any namespace by clicking on the details button to the right of that namespace.
```

![namespace_tab](/docs/workflow-components/namespace_tab.png)

> On clicking the details button for any namespace, you land on the namespace overview page which details the executions of flows in that namespace.

Reviewer: ```suggestion
When you select the details button for any namespace, the namespace overview page opens which details the executions of flows in that namespace.
```

![namespace_overview](/docs/workflow-components/namespace_overview.png)

On the top of this page, you have different tabs:

1. **Overview:** This is the default landing page of the Namespace. This page contains the dashboards and summary about the executions of different flows in this namespace.

> 2. **Editor:** This is the inbuilt editor wherein you can add/edit the namespaceFiles.

Reviewer: ```suggestion
2. **Editor:** the in-built editor where you can add/edit the namespaceFiles.
```

> 3. **Flows:** This page shows all the flows in the namespace. It gives a brief about each of the flows like the flow ID, labels, last execution date and last execution status, and the execution statistics. By clicking on the details button that is present to the rightmost of the flow, you can navigate to that flow's page.

Reviewer: ```suggestion
3. **Flows:** shows all the flows in the namespace. It gives a brief about each of the flows including the flow ID, labels, last execution date and last execution status, and the execution statistics. By selecting the details button on the right of the flow, you can navigate to that flow's page.
```

> 4. **KV Store:** On this tab, you can manage the key-values pairs associated with this namespace. More details on KV Store can be found [here](/concepts/kv-store).

Reviewer: ```suggestion
4. **KV Store:** manage the key-values pairs associated with this namespace. More details on KV Store can be found [here](/concepts/kv-store).
```
